README.md

```diff
@@ -54,52 +71,82 @@ brew install git     # if not already installed
 brew install asdf
 # then follow instructions to update ~/.zshrc and restart terminal
 brew install openssl readline sqlite3 xz zlib tcl-tk     # python dependencies
-asdf plugin-add python      # python plugin
-asdf list all python     # all the python versions available in asdf
-asdf install python 3.8.15
-asdf local python 3.8.15
-# creates a .tool-versions in current directory that shouldn't be checked in
-# OR `asdf global python 3.8.15` creates it in your $HOME
-python -V
-brew install poetry
-poetry env use 3.8      # to make sure poetry is using correct version of Python
-
-# INSTALL NODE with nvm
-brew install nvm
-mkdir ~/.nvm
-# follow instructions to add PATH info to shell profile to ~/.zshrc and restart terminal
-nvm install v16.14
 
 # INSTALL USEFUL THINGS
-brew install shellcheck jq
+brew install jq
 
-# INSTALL JAVA via SDKMan
-# Install SDKMan from https://sdkman.io/install, including .zshrc update
-curl -s "https://get.sdkman.io" | zsh
-source "/Users/<mac-username>/.sdkman/bin/sdkman-init.sh"
-sdk version
-sdk list java
-sdk install java 11.0.17-zulu
-sdk default java 11.0.17-zulu     # make default
+```
+
+### asdf setup
+
+You need to run the following to install the needed asdf packages
+
+```
+asdf plugin add python
+asdf plugin-add poetry https://github.com/asdf-community/asdf-poetry.git
+asdf plugin add shellcheck https://github.com/luizm/asdf-shellcheck.git
+asdf plugin add nodejs https://github.com/asdf-vm/asdf-nodejs.git
+asdf install
 ```
 
 ### Install packages
 
 Install the packages with `make install`, then verify everything is installed correctly by running the default `make` target.
 
-#### Pre-commit hooks
+</details>
+
+### Pre-commit hooks
 
-Some pre-commit hooks are installed as part of the install above to ensure you can't commit invalid spec changes by accident. A combination of these checks are also run in CI.
+Some pre-commit hooks are installed as part of the install above to ensure you can't commit invalid spec changes by accident and to run basic lint checks.  
+The pre-commit hook uses python package pre-commit and is configured in the file .pre-commit-config.yaml.  
+A combination of these checks are also run in CI.
 
 ### Environment Variables
 
 Various scripts and commands rely on environment variables being set. These are documented with the commands.
 
 :bulb: Consider using [direnv](https://direnv.net/) to manage your environment variables during development and maintaining your own `.envrc` file - the values of these variables will be specific to you and/or sensitive.
 
+The following are recommended to place in the .envrc file
+
+```
+export PACT_PROVIDER=nhsd-apim-eps
+export PACT_BROKER_BASIC_AUTH_PASSWORD=<SECRET>
```

Comment:
Any tips on where to obtain these <SECRET>s?

Reply:
will pass them to you - dont want to post on public forum about them